src/backoffice/settings/languages/workspace/language/language-workspace.element.ts

```diff
@@ -6,24 +6,33 @@ import { UmbLanguageWorkspaceContext } from './language-workspace.context';
 import { UmbRouterSlotInitEvent } from '@umbraco-cms/internal/router';
 import { UmbLitElement } from '@umbraco-cms/internal/lit-element';
 
+import './language-workspace-edit.element';
+
 @customElement('umb-language-workspace')
 export class UmbLanguageWorkspaceElement extends UmbLitElement {
 	static styles = [UUITextStyles, css``];
 
 	#languageWorkspaceContext = new UmbLanguageWorkspaceContext(this);
+	#element = document.createElement('umb-language-workspace-edit');
 
 	#routerPath? = '';
 
-	// TODO: add create route
 	@state()
 	_routes = [
 		{
 			path: 'edit/:isoCode',
-			component: () => import('./language-workspace-edit.element'),
```

Comment:
Didn't this work before or is there a reason to change this? Now we are loading the edit component every time you see the overview of languages.

Reply:
The idea was to not reload this element when going from create to edit, since they use the same layout the element is reused for both paths

Reply:
Does it reload, though, or does it reuse the already existing element?

Reply:
I believe it hits the already existing element provided, but @nielslyngsoe can probably shed some light on this :)

Reply:
This is the way we are doing it now for workspace routing in general — Seems alright for now, but we might look into when the element is loaded. :-)